### PR TITLE
Due to recent changes in React this technique to scroll to an element no longer works.

### LINKED ISSUE #3848: Cannot view machine view if machines exist
If you view the machine view when you have deployed machines there is a js error which breaks the app.

### PR DESCRIPTION
**Pull request by hatched**
Fixes #3848

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Open a model with machines and then switch to the machine view.

File: jujugui/static/gui/src/app/components/machine-view/machine-view.js [hunk 1]
```diff
@@ -229,42 +226,16 @@ class MachineView extends React.Component {
...
     Handle selecting a machine.
 
     @param {String} id The machine id to select. If the id is null, no machine
       is selected.
-    @param {Boolean} scrollToMachine Whether to scroll to the selected machine.
   */
-  selectMachine(id, scrollToMachine=true) {
-    if (id === null) {
+  selectMachine(id) {
+    if (!id || id === null) {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/machine-view/machine-view.js]: null evals to false so the second part of the OR is redundant.

Comment (hatched) [reply jujugui/static/gui/src/app/components/machine-view/machine-view.js]: You are correct!

**Comment by frankban** [comment]
QA is ok.
Looks good with one suggestion.

**Pull request merged**